### ISSUE TITLE
false negatives when reading single file

### ISSUE THREAD

**Issue by marionebl** [marionebl/jsonlint-cli#1]
### Actual

When reading invalid json files from fs by invokations such as `jsonlint-cli package.json` the linter exists with an code of 0.

``` json
{
   "name": "package"
```
### Expected

The linter should exit with code 1.
